CMakeLists.txt

```diff
@@ -44,11 +50,18 @@ if(USE_GME_NSFE AND NOT USE_GME_NSF)
     set(USE_GME_NSF 1 CACHE BOOL "Enable NES NSF music emulation" FORCE)
 endif()
 
-option(BUILD_SHARED_LIBS "Build shared library (set to OFF for static library)" ON)
-option(BUILD_FRAMEWORK "Build framework instead of dylib on macOS" OFF)
+if(DEFINED BUILD_SHARED_LIBS)
+  set(GME_BUILD_SHARED_DEFAULT ${BUILD_SHARED_LIBS})
+else()
+  set(GME_BUILD_SHARED_DEFAULT ON)
+endif()
```

Comment:
Please use 4-spaces indentation as other code does. Don't mix different indenting lengths.

Reply:
... anyway, there are some old code does the same, but I should fix it as soon as I will merge this.

Reply:
Ah, sorry about that, yes, some of the code in gme/CMakeLists.txt was 2-spaced around where I was editing so I got a little confused, I can update these too if you want.

Reply:
Ye, I want.